src/plugins/intel_gna/CMakeLists.txt

```diff
@@ -135,6 +135,13 @@ if(NOT BUILD_SHARED_LIBS)
     endif()
 endif()
 
-add_subdirectory(tests)
-
+# this is to prevent GNA failing on pipeline until new style will be finalizeda and implemented in sources.
+if(ENABLE_GNA_NCC)
```

Comment:
why do  we need GNA specific variable?

Reply:
Yes, because currently intel_gna plugin don't fully align with naming convention, so pipeline would fail.

Reply:
I am Ok to merge common logic, but suggest adding this functionality to GNA once GNA is compliant with NCC config

Reply:
I have removed gna specific condition and gna.style file in current PR.